src/routes/Settings/SettingsIndexView.js

```diff
@@ -176,7 +176,7 @@ class SettingsIndexView extends PureComponent<Props, State> {
     }).filter((elementProps) => {
       const { description, title, searchTags } = elementProps
 
-      if (description === null || description === false) {
+      if (description === null || description === '') {
```

Comment:
`if (!description)`?

Reply:
Why do you set it to empty string and then check for null?

Reply:
Are you sure it is a good idea to check setting display implicitly via its description, not via specific property?

Reply:
also
`if (description === null || description === '')` equals `if (!description)`